ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-nightly.yaml

```diff
@@ -36,7 +36,7 @@ tests:
     - chain: openshift-qe-cluster-density-v2
     workflow: rosa-aws-sts
 - as: perfscale-rosa-multiaz-cluster-density-v2-24nodes-periodic
-  cron: 0 12 * * 0
+  cron: 0 1 * * 0
```

Comment:
Similar to the ingress perf pr, is there a reason your changing every test cron? Its just changing it by an hour?

Reply:
For the cron jobs today, I could not find/notice a pattern
and I did not want to randomly add a new test job, as such scheduled the cron
in such a manner, each is scheduled to be after 3 hours.

Reply:
We should discuss this prior to changing it. In discussion with @svetsa-rh i understand they stagger cron runs to avoid running out of resources.